app/component/CityVehicleStopContent.js

```diff
@@ -8,14 +8,12 @@ import {
   getCitybikeCapacity,
   BIKEAVL_UNKNOWN,
   BIKEAVL_WITHMAX,
-  BIKESTATION_OFF,
-  BIKESTATION_CLOSED,
```

Comment:
These constant values can be removed from the citybikes.js (including BIKESTATION_ON also) as they are no longer used.